### ISSUE TITLE
Several menu items missing access keys

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#7482]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.4.656
    OS Version      : Windows (and some Linux desktops, I believe)
    R Version       : Any

### Steps to reproduce the problem
Bring up main menus via Alt key, and confirm that each menu item has an access key (an underlined letter).

### Describe the problem in detail
Following items are missing access keys:

Code / Soft Wrap Long Lines
Code / Rainbow Parentheses
View / Panes / Add Source Column
Session / Set Working Directory / To Source File Location
Tools / Show Command Palette
Help / Diagnostics / Show Keyboard Shortcut Commands

### Describe the behavior you expected
Access keys for all menu items

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by R-Hannibal** [comment]
Verified Fixed on: RStudio Version - 1.4.739, R Version - 4.0.2, Windows 10 x64 - Desktop 

All listed missing Menu Access Keys have been successfully added and seem operable.
